src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/FeedbackControllerTests.cs

```diff
@@ -0,0 +1,70 @@
+#nullable enable
+using FluentValidation.TestHelper;
+using GovUk.Education.ExploreEducationStatistics.Common;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Requests;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Tests.Controllers;
+
+public class FeedbackControllerTests(TestApplicationFactory testApp) : IntegrationTestFixture(testApp)
+{
+    private const string BaseUrl = "api/feedback";
+
+    [Fact]
+    public async Task CreateFeedback_Success()
+    {
+        // Arrange
+        var client = TestApp.CreateClient();
+
+        var request = new FeedbackCreateRequest
+        {
+            Response = FeedbackResponse.ProblemEncountered,
+            Url = "/",
+            UserAgent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36",
+            Context = "What were you doing?",
+            Intent = "What did you hope to achieve?",
+            Issue = "What went wrong?",
+        };
+
+        // Act
+        await client.PostAsync(BaseUrl, JsonContent.Create(request));
+
+        // Assert
+        await using var context = TestApp.GetDbContext<ContentDbContext>();
+        var saved = await context.Feedback.FirstAsync();
+
+        Assert.Equivalent(request, saved);
+    }
+
+    [Fact]
+    public async Task FeedbackCreateRequestValidator_InvalidFields_IncludesRelevantErrors()
```

Comment:
Would check validation is working via an integration test to ensure everything is working correctly. It's pretty straightforward to do this as you can just `PostAsync` an invalid request and assert on the response.